src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int figureNumber = 11;
+
+        int figuresNumberInSetOne = 0;
+
+        if (figureNumber % 2 == 0) {

```

Comment:
The calculation of 'figuresNumberInSetOne' can be simplified by using the ceiling operation of the division of 'figureNumber' by 2. This will eliminate the need for the if-else condition.